pkg/runner/eventmanager.go

```diff
@@ -89,12 +90,50 @@ func (c *EventManager) Manage(id string, stdout, stderr io.ReadCloser) {
 	go func() {
 		defer stderr.Close()
 		defer c.wg.Done()
-		scanner := bufio.NewScanner(stderr)
-		for scanner.Scan() {
-			c.msg(idx, id, time.Now(), Error, scanner.Text())
-		}
-		if err := scanner.Err(); err != nil {
-			c.msg(idx, id, time.Now(), Error, "stderr error: "+err.Error())
+
+		decoder := json.NewDecoder(stderr)
+		for {
+			event := make(map[string]json.RawMessage, 16)
+			if err := decoder.Decode(&event); err != nil {
```

Comment:
Unfortunately, I can't find a _nice_ way to parse log events with arbitrary fields.